api/models.py

```diff
@@ -191,30 +209,22 @@ class Node(DatabaseModel):
         description='Dictionary with names mapping to node associated \
 URLs (e.g. URL to binaries or logs)'
     )
-    created: Optional[datetime] = Field(
+    created: datetime = Field(
         default_factory=datetime.utcnow,
         description='Timestamp of node creation'
     )
-    updated: Optional[datetime] = Field(
+    updated: datetime = Field(
         default_factory=datetime.utcnow,
         description='Timestamp when node was last updated'
     )
-    timeout: Optional[datetime] = Field(
+    timeout: datetime = Field(
+        default_factory=DefaultTimeout(hours=6).get_timeout,
```

Comment:
Is there any specific reason to set 6 hrs as default?

Reply:
Because less than that is too short and longer than that is too long...  Ideally the default should never be used, but if it's used then that's the kind of timeframe someone would expect results to arrive after a new kernel revision has been pushed to git.